qdrant-landing/content/documentation/guides/migration.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Migration
+weight: 10
+aliases:
+  - ../migration
+---
+
+
+Migrating data between vector databases, especially across regions, platforms, or deployment types, can be a hassle. That’s where the **Qdrant Migration Tool** comes in. The tool supports all your migration needs, including moving from a self-hosted instance to Qdrant and switching cloud providers.  
+
+In this tutorial, we will learn how to use the migration tool and walk through a practical example of migrating from other vector databases to Qdrant. 
```

Comment:
You can also use this to migrate data between Qdrant instances, which is especially useful when migrating from Qdrant OSS to Qdrant Cloud.

Reply:
Added